.codespellignorelines

```diff
@@ -1,2 +1,3 @@
                 input="thsi si a spelilgn imstkae.",
-
+            input="thsi si a spelilgn imstkae.",
```

Comment:
nit: do we need to edit this file.

Reply:
Nope, unfortunately this was multiple separate instances of this spelling "error" being used in `test_openai.py` at different indentation levels, and codespell ignores files require exact lines (including whitespace).